packages/profiling-node/src/integration.ts

```diff
@@ -13,7 +13,7 @@ import {
   spanToJSON,
```

Comment:
This contains the tests that were previously in spanProfileUtils, it made no sense to call it that as it was actually testing the integration.